snuba/datasets/message_filters.py

```diff
@@ -57,6 +84,10 @@ class KafkaHeaderSelectFilter(StreamMessageFilter[KafkaPayload]):
     header_key: str
     header_value: str
 
+    @classmethod
+    def config_key(cls) -> str:
+        return "KafkaHeaderSelectFilter"
```

Comment:
we can just set the `config_key` in the superclass to `cls.__name__`

Reply:
That was my initial thought as well, but wouldn't this decrease the grep-ability?

Reply:
no because you can grep the class name and get the class